zproject/settings.py

```diff
@@ -772,6 +772,7 @@ def get_secret(key):
             'js/common.js',
             'node_modules/moment/moment.js',
             'node_modules/moment-timezone/builds/moment-timezone-with-data.js',
+            'js/rtl.js',
```

Comment:
I think we should be adding this to `app`, not `common`, since we don't have plans to use it on the logged-out pages.